aiidalab/utils.py

```diff
@@ -29,45 +30,30 @@
 logger = logging.getLogger(__name__)
 FIND_INSTALLED_PACKAGES_CACHE = TTLCache(maxsize=32, ttl=60)  # type: ignore
 
-# Warning: try-except is a fix for Quantum Mobile release v19.03.0 where
-# requests_cache is not installed.
-try:
-    import requests_cache
```

Comment:
Does this problem not exist in Quantum Mobile anymore?

Reply:
This code is 3 years old, so I would certainly hope so. :-) Would you be able to check with the latest version of Quantum Mobile? If that is too much work, I can put this back if it is of concern. I don't actually know how QM is build and how aiidalab is integrated into it, but I would hope that the dependencies of this package are correctly installed.

Reply:
I think @unkcpz works on the latest QM. Maybe he can comment when he gets time. For now, I would suggest putting it back.

Reply:
Okay, I added it back and opened https://github.com/aiidalab/aiidalab/issues/452 as a follow-up.

Unlike in the old code, the code path that does not use requests_cache uses the `requests.Session` object, which should give much better performance as well since the HTTP connection should be preserved between calls.